src/framework/mpe/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: GPL-3.0-only
+# MuseScore-CLA-applies
+#
+# MuseScore
+# Music Composition & Notation
+#
+# Copyright (C) 2021 MuseScore BVBA and others
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+set(MODULE mpe)
+
+set(MODULE_SRC
+    ${CMAKE_CURRENT_LIST_DIR}/events.h
+    ${CMAKE_CURRENT_LIST_DIR}/mpetypes.h
+    )
+
+set(MODULE_QRC mpe.qrc)
```

Comment:
I believe it's better to add `dummy.cpp` instead of an empty resource file :)

Reply:
Only that there is no such file?
Same is true BTW for src/framework/telemetry/CMakeLists.cpp and ...telemetry.qrc
Both empty qrc files lead to a build warning "No resources in ..."

Reply:
I just prepared this along the way. The actual resources would be introduced at on of the next PRs